### PR TITLE
Replace "foo" string with a less generic one

### PR DESCRIPTION
**Pull request by israel-hdez**
If template parsing fails, the "foo" string would appear in pod logs. This should be very hard to happen, since the template is built-in. Changing, simply, to have more meaningful logs.

Fixes opendatahub-io/modelmesh-serving#114

### CODE REVIEW

**Comment by spolti** [comment]
@israel-hdez it might need a rebase.

File: controllers/config/templating.go
Comment (spolti) [inline controllers/config/templating.go:52]: maybe `ModelMeshTemplate`?

Comment (israel-hdez) [reply controllers/config/templating.go]: It is fixed.

**Review by rafvasq — APPROVED** [review]

**Comment by israel-hdez** [comment]
I think somebody needs to also say `/lgtm`.

**Comment by rafvasq** [comment]
> I think somebody needs to also say `/lgtm`.

I just wanted to give @spolti the final say since he originally requested changes.

**Comment by israel-hdez** [comment]
> > I think somebody needs to also say `/lgtm`.
> 
> I just wanted to give @spolti the final say since he originally requested changes.

He is on vacation right now.
This is not urgent, so we can wait :slightly_smiling_face:

**Comment by israel-hdez** [comment]
Thanks, @rafvasq .

**Pull request merged**
